Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -153,9 +154,11 @@ void RemoteRenderingBackend::workQueueInitialize()
 
 void RemoteRenderingBackend::workQueueUninitialize()
 {
+    Locker locker { m_lock };
+    m_remoteImageBuffers.clear();
```

Comment:
Can we add an additional scope to drop the lock after m_remoteImageBuffers is done clearing?